src/styles/fonts.css

```diff
@@ -0,0 +1,11 @@
+@font-face {
+  font-family: Avenir;
+  src: url('../fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('../fonts/Avenir-Heavy.ttf');
+  font-weight: 400;
```

Comment:
The `font-weight` for Avenir-Heavy should be 800 or 900, as 'Heavy' is typically associated with these weight values. The current value of 400 is usually used for 'Regular' or 'Normal' font weights.